packages/purpose-process/test/testGetRiskAnalysisConfigurationByVersion.ts

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
+import {
+  getMockEService,
+  getMockTenant,
+  randomArrayItem,
+  writeInReadmodel,
+} from "pagopa-interop-commons-test/index.js";
+import {
+  EServiceId,
+  TenantId,
+  eserviceMode,
+  generateId,
+  tenantKind,
+  toReadModelEService,
+} from "pagopa-interop-models";
+import { describe, expect, it } from "vitest";
+import { genericLogger, getFormRulesByVersion } from "pagopa-interop-commons";
+import {
+  riskAnalysisConfigVersionNotFound,
+  eserviceNotFound,
+  tenantKindNotFound,
+  tenantNotFound,
+} from "../src/model/domain/errors.js";
+import {
+  eservices,
+  purposeService,
+  tenants,
+} from "./purposeService.integration.test.js";
+export const testGetRiskAnalysisConfigurationByVersion = (): ReturnType<
+  typeof describe
+> =>
+  describe("retrieveRiskAnalysisConfigurationByVersion", async () => {
+    it("should retrieve risk analysis configuration by version (Eservice mode: deliver)", async () => {
```

Comment:
In general, we could highlight a little bit (both in the test title and in the test implementation) the fact that the eservice mode determines which tenant has to be used to retrieve the risk analysis configuration according to its `kind`.